catkin_ws/src/sensors/package.xml

```diff
@@ -40,5 +42,7 @@
   <exec_depend>message_runtime</exec_depend>
   <exec_depend>python-bitstring-pip</exec_depend>
   <exec_depend>python3-serial</exec_depend>
+  <exec_depend>sensor_msgs</exec_depend>
+  <exec_depend>python3-numpy</exec_depend>
```

Comment:
I've never seen us have to put python3-numpy, did you do this because it gave u an error or smth? Or can we remove

Reply:
I kept getting an error related to the initialization of a np.quaternion object. So, when I looked at other packages that also use np.quaternion (sim, controls, state estimation, ...), they all had python3-numpy in their package.xml. I'm not 100% sure if this was the solution considering that I changed a bunch of things, but the error went away.

Reply:
ive had a similar problem in the past that was related to the order of importing numpy/quaternion (first numpy then quaternion)